pipeline_steps/deploy.groovy

```diff
@@ -55,6 +68,13 @@ def upgrade(String stage_name, String upgrade_script, List env_vars) {
           sh "git reset --hard"
         }
         common.prepareRpcGit()
+        if ( stage_name == "Leapfrog Upgrade" ) {
+          create_test_resource_load(
+            env.LEAPFROG_TEST_SERVERS as Integer,
```

Comment:
Could be used for intra-series or will be? When is that likely to happen? I used leapfrog in the variable names because, while the yaml file requires they be specified as parameters for all jobs, they only have meaning for leapfrog upgrades. Using specific variable names should prevent any confusion regarding their use and given the complexity of this repo I think anything that makes the code more readable is a benefit.

If at some point in the future this functionality does get used more generally, there is nothing to prevent the names being changed. Given that we are not committing to this variables being stable, I do not believe this causes an issue.